source/entities/Creature.cpp

```diff
@@ -1887,8 +1887,7 @@ void Creature::createStatsWindow()
                                         CEGUI::Event::Subscriber(&Creature::CloseStatsWindow, this));
 
     // Set the window title
-    childWindow = mStatsWindow->getChild("__auto_titlebar__");
-    childWindow->setText(getName() + " (" + getDefinition()->getClassName() + ")");
+    mStatsWindow->setText(getName() + " (" + getDefinition()->getClassName() + ")");
```

Comment:
Small git-fu notice: whenever possible, try to split git commit logs into a short title + a more lengthy description after a newline, e.g.:

```
Change creature window stats title by using setText

Previously, it was addressing built-in window which caused a crash when using CEGUI 0.8.5.
```

That would prevent a broken long title as we have here.